src/nft/components/details/detailsV2/ActivityTableContent.tsx

```diff
@@ -1,3 +1,5 @@
+import { TableContentHeight } from './DataPageTable'
+
 export const ActivityTableContent = () => {
-  return <div>Activity Content</div>
+  return <div style={{ height: `${TableContentHeight}px` }}>Activity Content</div>
```

Comment:
this should just be a styled component - I don't know if `TableContentHeight` needs to be a constant in the first place

Reply:
This is just a temporary height set for visualization purposes and will be removed by the implementing engineer for that component. I made it a const for consistency across the tabs and to reduce the number of magic number heights